settingup.tex

```diff
@@ -108,8 +108,8 @@ \subsection{Replacing the Real-Time Clock Battery}
 battery socket \textit{away} from the battery to remove it. Next, insert the new battery with the side labelled
 {\bf + facing up}, and press it into place.
 
-Once you have re-assembled your MEGA65, you can set the time in the Configure menu. You can read more about setting
-the time on page \pageref{configuring-chipset}.
+Once you have re-assembled your MEGA65, you can set the time in the Configure menu. For more information on how to set
+the Real-Time Clock, refer to the Configuring Utility section on page \pageref{sec:configuration-utility}.
```

Comment:
typo, should be: ...refer to the **_Configuration_** Utility...

Reply:
fixed